src/model/gameplay.test.js

```diff
@@ -0,0 +1,83 @@
+import Gameplay from './Gameplay';
+
+describe('gameplay', () => {
+  afterEach(() => {
+    jest.clearAllTimers();
+    jest.useRealTimers();
+  });
+
+  it('should properly return game object', () => {
+    const gameplayObject = Gameplay();
+    expect(typeof gameplayObject).toBe('object');
+    const expectedToReceiveKeys = [
+      'currentQuestion',
+      'questionHistory',
+      'gameMode',
+      'gamePlayer',
+      'playerName',
+      'score',
+      'endGame',
+      'onAnswerCheck',
+      'startGame',
+      'startTiming',
+      'questionIndex',
+      'measureGameTime',
+    ].sort();
+    const receivedGameKeys = Object.keys(gameplayObject).sort();
+    expect(JSON.stringify(receivedGameKeys) === JSON.stringify(expectedToReceiveKeys)).toBe(true);
```

Comment:
To jest moim zdaniem nieczytelne i jakbyś wróciła do tego po np. roku, nie wiedziała byś o co Ci chodziło. 
Proponuję: 
```suggestion
    expectedToReceiveKeys.forEach(key => {
      expect(receivedGameKeys).toContains(key);
    })
```